### PR TITLE
feat: add check for service bindings

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
Still a Work in Progress while @wtrocki tests it out.

File: consumer-backend/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "consumer.js",
   "scripts": {
     "lint": "eslint .",
-    "openshift": "npx nodeshift --dockerImage=registry.access.redhat.com/ubi8/nodejs-14",
+    "openshift": "npx nodeshift --dockerImage=registry.access.redhat.com/ubi8/nodejs-12",
```
Comment (lholmquist) [inline consumer-backend/package.json:8]: any reason for the ndoe version downgrade?

**Comment by wtrocki** [comment]
This works well with Strimzi but we need SASL/Plain Auth for this

File: consumer-backend/package.json (hunk 1 above)
Comment (mhdawson) [reply consumer-backend/package.json:8]: I probably should have mentioned : https://bugzilla.redhat.com/show_bug.cgi?id=1950018.  installing from Git seems broken in the 14.x container.

**Comment by mhdawson** [comment]
@wtrocki, what do we have to do for "SASL/Plain Auth ".  If I had more info or a way to test that we could figure out what to adjust.

File: consumer-backend/package.json (hunk 1 above)
Comment (mhdawson) [reply consumer-backend/package.json:8]: Seems like we are also missing openssl headers in the 14.x containers, so for node-rdkafka we should leave this as nodejs-12 for now.

**Comment by mhdawson** [comment]
Worked with @wtrocki to test with RH managed Kafka, landing

**Comment by mhdawson** [comment]
Luke had taken a look, going to test after rebase on latest changes and then land.

**Pull request merged**